docs/src/guides/tables.md

```diff
@@ -308,7 +304,9 @@ Use the `delete()` method on tables to delete rows from a table. To choose which
     import lancedb
     import pandas as pd
 
-    data = pd.DataFrame({"x": [1, 2, 3], "vector": [[1, 2], [3, 4], [5, 6]]})
+    data = pd.DataFrame([{"x": 1, "vector": [1, 2]},
+                         {"x": 2, "vector": [3, 4]},
+                         {"x": 3, "vector": [5, 6]}])
```

Comment:
My intention with being row-based is we would completely remove pandas from those examples. So something like:
```suggestion
    data = [{"x": 1, "vector": [1, 2]},
            {"x": 2, "vector": [3, 4]},
            {"x": 3, "vector": [5, 6]}]
```